tests/plugins/autogrow/autogrowwidth.js

```diff
@@ -0,0 +1,37 @@
+/* bender-tags: editor, autogrow */
+/* bender-ckeditor-plugins: autogrow */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	bender.test( {
+		init: function() {
+			// Remove root editor element border.
+			var editorRootElement = CKEDITOR.document.getById( 'cke_test_editor' );
+			editorRootElement.setStyle( 'border', 'none' );
+		},
+
+		// #4286
+		'test editor width': function() {
```

Comment:
Since there are no test for `autogrow` at all, could we add similar test as this one but without removing border (checking if width doesn't change and if height grows)?